src/main/java/com/microfocus/application/automation/tools/octane/model/processors/projects/AbstractProjectProcessor.java

```diff
@@ -209,6 +211,7 @@ public CIBuildStatusInfo getBuildStatus(String paramName, String paramValue) {
 							} else {
 								status.setBuildStatus(CIBuildStatus.FINISHED);
 								status.setResult(BuildHandlerUtils.translateRunResult(aBuild));
+								status.setEnvironmentOutputtedParameters(getOutputEnvironmentParams(aBuild));
```

Comment:
Same for here, please use the helper inside the set method, it more readable like other setters instead of adding it on import

Reply:
done